src/View/mainpage/WindowingSlider.py

```diff
@@ -32,15 +32,20 @@ class WindowingSlider(QWidget):
     MAX_CLICK_DIST = 5
     # Minimum rendering height for bottom bar
     MIN_BOTTOM_INDEX = 4
+    # Window/Level consts
+    MAX_PIXEL_VALUE = 4096
+    LEVEL_OFFSET = 1000
```

Comment:
why is there a level offset?  and why is the value 1000?  Perhaps this has to do with the values retrieved from Qt for the slider bars?